src/components/templates/AnimateMap/game/map/entities/EntityFactory.ts

```diff
@@ -356,7 +356,7 @@ export default class EntityFactory {
     entity
       .add(new ArtcarComponent(user, fsm))
       .add(new PositionComponent(user.x, user.y, 0, scale, scale))
-      .add(new HoverableSpriteComponent());
+      .add(new HoverableSpriteComponent(undefined, undefined, true));
```

Comment:
[OOS]

The problems with positional parameters and boolean traps. In some future refactor this might be:
```ts
.add(new HoverableSpriteComponent({ isCursorPointer:true }));
```